tests/baselines/reference/declarationEmitSpreadStringlyKeyedEnum.js

```diff
@@ -0,0 +1,53 @@
+//// [declarationEmitSpreadStringlyKeyedEnum.ts]
+enum AgeGroups { "0-17" , "18-22" , "23-27" , "28-34" , "35-44" , "45-59" , "60-150" }
+export const SpotifyAgeGroupEnum = { ...AgeGroups };
+
+//// [declarationEmitSpreadStringlyKeyedEnum.js]
+"use strict";
+var __assign = (this && this.__assign) || function () {
+    __assign = Object.assign || function(t) {
+        for (var s, i = 1, n = arguments.length; i < n; i++) {
+            s = arguments[i];
+            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
+                t[p] = s[p];
+        }
+        return t;
+    };
+    return __assign.apply(this, arguments);
+};
+exports.__esModule = true;
+exports.SpotifyAgeGroupEnum = void 0;
+var AgeGroups;
+(function (AgeGroups) {
+    AgeGroups[AgeGroups["0-17"] = 0] = "0-17";
+    AgeGroups[AgeGroups["18-22"] = 1] = "18-22";
+    AgeGroups[AgeGroups["23-27"] = 2] = "23-27";
+    AgeGroups[AgeGroups["28-34"] = 3] = "28-34";
+    AgeGroups[AgeGroups["35-44"] = 4] = "35-44";
+    AgeGroups[AgeGroups["45-59"] = 5] = "45-59";
+    AgeGroups[AgeGroups["60-150"] = 6] = "60-150";
+})(AgeGroups || (AgeGroups = {}));
+exports.SpotifyAgeGroupEnum = __assign({}, AgeGroups);
+
+
+//// [declarationEmitSpreadStringlyKeyedEnum.d.ts]
+declare enum AgeGroups {

```

Comment:
Another test for const enum ?

Reply:
`'const' enums can only be used in property or index access expressions or the right hand side of an import declaration or export assignment or type query.` - it can't be spread.